### DISCUSSION TITLE
Domains gone from Hobby account after Vercel migration to hobby teams

### DISCUSSION THREAD

**Discussion by namadaza** [vercel/community#6300] [Help]
### Summary

I've had all my domains visible on my hobby account. When Vercel made the [change](https://vercel.com/changelog/2024-01-account-changes) for all hobby accounts to be free teams, I no longer see any of my domains on my dashboard.

canogadigital.com
samueljcomroe.com
typetea.com
arabicbridge.com

I've verified for each domain that nameservers haven't changed on my registrar site (namecheap).

### Steps to Reproduce

1. Vercel.com dashboard
2. "amanazad's Projects"
3. Click on "Domains"
4. See no domains listed on page

**Comment by amyegan** [comment]
Hey @namadaza. I sounds like your account must have been updated to our new structure in the last few days. If that's the case, your domains will still be listed under your personal account and can be moved to your new hobby team. 

You can learn more about it in https://github.com/vercel/community/discussions/6298
